packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -75,11 +128,15 @@ export class CustomScrollbar extends Component<Props> {
       passedProps.style.display = 'none';
     }
 
-    return <div {...passedProps} className={track} />;
+    return (
+      <div {...passedProps} className={track === 'track-vertical' ? styles.trackVertical : styles.trackHorizontal} />
+    );
   };
 
-  renderThumb = (thumb: 'thumb-horizontal' | 'thumb-vertical', passedProps: any) => {
-    return <div {...passedProps} className={thumb} />;
+  renderThumb = (thumb: 'thumb-vertical' | 'thumb-horizontal', passedProps: any) => {
+    return (
+      <div {...passedProps} className={thumb === 'thumb-vertical' ? styles.thumbVertical : styles.thumbHorizontal} />
+    );
```

Comment:
Instead of passing the string argument `thumb` ('thumb-vertical' | 'thumb-horizontal') shouldn't we pass the correct className instead? This would eliminate the ternary in className:
`className={thumb === 'thumb-vertical' ? styles.thumbVertical : styles.thumbHorizontal}`

Reply:
Thinking more about this, I don't see why `renderThumb` exists at all. Maybe just move this code to `renderThumbHorizontal` and `renderThumbVertical` instead?